### DISCUSSION TITLE
【内有图和详细描述】NATFRP无法穿透SRCDS的27015~27019端口

### DISCUSSION THREAD

**Discussion by leodenglovescode** [natfrp/wiki#249] [使用问题]
开始我想穿透我的胜利之日：起源（Day of Defeat: Source）服务器的，因为所有Source游戏端的服务器界面只会扫描27015~27019端口，我也想别人能发现我的服务器，所以我就这么设置了一下：

![image](https://user-images.githubusercontent.com/71826908/210379193-eedf6c1a-c446-42b6-9bcd-bb06e93a6861.png)

完全是按照教程来的，服务器开的是这个端口：

![image2](https://user-images.githubusercontent.com/71826908/210379506-d9ef8735-87e8-411a-8f74-ef53a9a2d027.png)

然鹅我的游戏端怎么都连不上，朋友也试了，不行:

![image3](https://user-images.githubusercontent.com/71826908/210379647-fc6992f4-65af-46f1-a3f0-14964e69ee81.png)

![image4](https://user-images.githubusercontent.com/71826908/210379697-2102335c-d547-4c00-9672-5bc4933abc91.png)

但是神奇的是，如果NATFRP的外部端口设定为自动生成，再把服务器端口调一下，就能连上了，哪个路过的好心人
帮帮我吧，快崩溃了。。。。。 :((((( 

谢谢嗷 Thanks♪(･ω･)ﾉ

**Comment by fakeboboliu** [comment]
![图片](https://user-images.githubusercontent.com/7552030/210583696-9c169f8c-1401-47d1-b62f-b6e6db4b9add.png)
